drivers/focuser/steeldrive.cpp

```diff
@@ -987,7 +987,7 @@ bool SteelDrive::startMotion(FocusDirection dir)
 
     // inward  --> decreasing value --> DOWN
     // outward --> increasing value --> UP
-    strncpy(cmd, (dir == FOCUS_INWARD) ? ":F2MDOW0#" : ":F1MUP00#", STEELDRIVE_CMD);
```

Comment:
Shouldn't we just changed STEELDRIVE_CMD to 10 , and then change in tty_write to STEELDRIVE_CMD - 1 ? I presume this affects other commands as well?